### ISSUE TITLE
Follow best practices for colour blind friendly visualisations

### ISSUE THREAD

**Issue by adonath** [gammapy/gammapy#3079]
CTA released a document describing best practices for data visualisation in a way friendly to color-blind people: https://www.cta-observatory.org/wp-content/uploads/2020/10/CTA_ColourBlindness_BestPractices2.pdf We should evaluate and adapt our default visualisations in Gammapy to follow these best practices.

**Comment by AtreyeeS** [comment]
Looking at the document, I see that the suggestion is to use `tableau- coloblind10` for colorcycles, and `viridis` (or `cividis`) for colormaps. Does this have to be done for each plot separately, or is there a way to set it as a default matplotlib argument so that it works globally?

**Comment by adonath** [comment]
@AtreyeeS Yes, we could just introduce a global Gammapy plotting style. E.g. via https://matplotlib.org/stable/tutorials/introductory/customizing.html#using-style-sheets

**Comment by bkhelifi** [comment]
As a simple 
```
import matplotlib.style as style
style.use('tableau-colorblind10')
```
can be used, I am wondering if the add of an HowTo is not enough....

**Comment by registerrier** [comment]
The addition of an howto showing how to use matplotlib colorblind themes is implemented in #4043.

A more complete solution would require modifying `plot` and `peek` methods to use different line styles.

Closing for now. If you think we should aim at using different markers or line styles, please re-open @adonath .
